cf_units/__init__.py

```diff
@@ -365,6 +369,13 @@ def date2num(date, unit, calendar):
         ...               cf_units.CALENDAR_STANDARD)
         array([6.5, 7.5])
 
+        # Integer type preferentially returned if possible:
```

Comment:
```suggestion
        >>> # Integer type preferentially returned if possible:
```

Now I got an html build to work, I find that this doesn't render correclty
(and neither does the original in #467 !)

It seems not to like the blank line + stops rendering as a code example.
If you prefix with the correctly indented ">>> ", that seems to fix it.

Reply:
Oh, on a roll here !!
I think you can use "<BLANKLINE>" somehow : see [here](https://www.sphinx-doc.org/en/master/usage/extensions/doctest.html#directive-doctest)

Reply:
Ah yes - I see now that this is doctest expecting the comment line to be an output of the previous test.
Originally, I was running `make doctest` to get Sphinx to run the doctests and this was passing (even though it shouldn't which makes me think it wasn't __really__ running the doctests). 😲

Now I am running the doctests with `pytest --doctest-modules` which correctly catches this error.

I believe that `<BLANKLINE>` is only valid when you **expect** a blank line as a test output, so I think the best solution here is to simply ensure the comment and any blank lines are prepended with  `>>>`. However, for styling consistency, I have removed the blank line (but kept the comment)

I've done this for both the `cftime.date2num` and `cftime.Unit.date2num`